ql.mm

```diff
@@ -9875,7 +9375,11 @@ Kalmbach axioms (soundness proofs) that require WOML
 $)
 
   $( New equation that holds in Hilbert space, discovered by Pavicic and Megill
-     (unpublished). $)
+     (unpublished).
+
+     [Editor's note:  The date this was added is unknown within the range
+     1997 to 29-Oct-2005. The axiom was discovered before 9-Aug-2003.]
+     (Contributed by NM, 1-Jan-1998.) $)
```

Comment:
```suggestion
     1997 to 9-Aug-2003.]  (Contributed by NM, 1-Jan-1998.) $)
```

Reply:
@digama0 if it's good for you, I'll let you apply and merge.

I'll add the bibliographic info I gave above in a future MR.